google-cloud-firestore/src/main/java/com/google/cloud/firestore/BulkWriterOptions.java

```diff
@@ -20,15 +20,31 @@
 
 /** Options used to disable request throttling in BulkWriter. */
 final class BulkWriterOptions {
```

Comment:
I think it might be time to make this an actual Builder. You can use AutoValue to reduce the amount of code you have to write. See https://github.com/googleapis/java-firestore/blob/edaa5395be0353fb261d954429c624623bc4e346/google-cloud-firestore/src/main/java/com/google/cloud/firestore/Query.java#L228

Reply:
attempted.